device-types/HPE/Aruba-EC-M.yaml

```diff
@@ -1,47 +1,48 @@
 ---
 manufacturer: HPE
-model: Aruba ec-m
+model: Aruba EC-M
 slug: hpe-aruba-ec-m
 part_number: '200890'
 u_height: 1
 is_full_depth: true
 airflow: front-to-rear
+weight: 26
+weight_unit: lb
 console-ports:
   - name: console
     type: de-9
     label: console
 power-ports:
```

Comment:
Hi Thank you for your contribution.
Are these modular PSUs? If so they should be modeled as `module-bays` with the required PSUs added to the modules folder if they don't already exist.